climada/hazard/base.py

```diff
@@ -818,6 +818,7 @@ def concat(cls, haz_list):
             return cls()
         haz_concat = haz_list[0].__class__()
         haz_concat.haz_type = haz_list[0].haz_type
+        haz_concat.centroids = Centroids(lat=[], lon=[], crs=haz_list[0].centroids.crs)
```

Comment:
Exactly! 😄 
Alternatively one may inject the centroids as an argument of `haz_list[0].__class__`. (And possibly skip the `haz_type` assignment. I suspect that it is already assigned at that point through the constructor)